pkg/resourcemanager/controller/tokenrequestor/reconciler.go

```diff
@@ -141,33 +140,37 @@ func (r *reconciler) reconcileServiceAccount(ctx context.Context, secret *corev1
 	return serviceAccount, nil
 }
 
-func (r *reconciler) reconcileSecret(ctx context.Context, sourceSecret *corev1.Secret, token string, renewDuration time.Duration) error {
+func (r *reconciler) reconcileSecret(ctx context.Context, log logr.Logger, sourceSecret *corev1.Secret, token string, renewDuration time.Duration) error {
 	patch := client.MergeFrom(sourceSecret.DeepCopy())
 	metav1.SetMetaDataAnnotation(&sourceSecret.ObjectMeta, resourcesv1alpha1.ServiceAccountTokenRenewTimestamp, r.clock.Now().UTC().Add(renewDuration).Format(time.RFC3339))
 
 	if targetSecret := getTargetSecretFromAnnotations(sourceSecret.Annotations); targetSecret != nil {
-		if _, err := controllerutil.CreateOrUpdate(ctx, r.targetClient, targetSecret, r.populateToken(targetSecret, token)); err != nil {
+		log.Info("Populating the token to the target secret", "targetSecret", client.ObjectKeyFromObject(targetSecret))
```

Comment:
Does it make sense to add a log entry for the depopulate token operation from the source secret that occurs few lines below?